BackEnd/routes.js

```diff
@@ -41,7 +41,7 @@ function checkTargetInSafezone(player, target, callback) {
 	Safezone.findOne({_id: target.mySafeZoneId}, (err, safezone) => {
 		if(!safezone) {
 			console.log("couldn't find safezone")
-			return false;
+			// return false;
```

Comment:
Shouldn't we still call `return;` here so that the rest of the code doesn't keep executing?

Reply:
no because they could be in the game central safe zone and not their individual one. Returning false would make us exit tooe arly.